velox/connectors/hive/SplitReader.cpp

```diff
@@ -141,6 +141,101 @@ void SplitReader::configureReaderOptions(
 void SplitReader::prepareSplit(
     std::shared_ptr<common::MetadataFilter> metadataFilter,
     dwio::common::RuntimeStatistics& runtimeStats) {
+  createReader();
+
+  if (testEmptySplit(runtimeStats)) {
+    return;
+  }
+
+  createRowReader(metadataFilter);
+}
+
+uint64_t SplitReader::next(uint64_t size, VectorPtr& output) {
```

Comment:
Are you changing anything in these methods?  Maybe put them to their original position so it is more clear what did you change

Reply:
> Are you changing anything in these methods? Maybe put them to their original position so it is more clear what did you change

Sure, I restored the orginal order in this commit, but added a second commit to just reorder them so that the order aligns with their declarations, which is in (access level desc, call order) order.